tests/integration/test_attach_backup_from_s3_plain/test.py

```diff
@@ -41,16 +41,17 @@ def test_attach_part(table_name, backup_name, storage_policy, min_bytes_for_wide
 
     create database ordinary_db engine=Ordinary;
 
-    create table ordinary_db.{table_name} engine=MergeTree() order by tuple() as select * from numbers(100);
+    create table ordinary_db.{table_name} engine=MergeTree() order by key partition by part as select number%5 part, number key from numbers(100);
     -- NOTE: name of backup ("backup") is significant.
-    backup table ordinary_db.{table_name} TO Disk('backup_disk_s3_plain', '{backup_name}');
+    backup table ordinary_db.{table_name} TO Disk('backup_disk_s3_plain', '{backup_name}') settings deduplicate_files=0;
 
     drop table ordinary_db.{table_name};
-    attach table ordinary_db.{table_name} (number UInt64)
+    attach table ordinary_db.{table_name} (part UInt8, key UInt64)
     engine=MergeTree()
-    order by tuple()
+    order by key partition by part
     settings
         min_bytes_for_wide_part={min_bytes_for_wide_part},
+        max_suspicious_broken_parts=0,
         storage_policy='{storage_policy}';
```

Comment:
Did you check empty files in backup somewhere?

Reply:
No, since this table does not have empty files.